src/index.html

```diff
@@ -18,65 +18,95 @@
     <link rel="icon" type="image/png" href="/img/favicon.png"/>
     <title ng-bind="title + ' - H.S.A. Confluente'">H.S.A. Confluente</title>
 
+    <!-- Script to make the navbar hoverable in desktop format -->
+    <script>
+        $('.dropdown-toggle').click(function(e) {
```

Comment:
What does this script add exactly? All browsers don't see a '#' as a link to a new page in my experience 😬